core/sessions/ldapauth/ldap.go

```diff
@@ -53,7 +54,7 @@ var ErrUserNotInUpstream = errors.New("LDAP query returned no matching users")
 var ErrUserNoLDAPGroups = errors.New("user present in directory, but matching no role groups assigned")
 
 type ldapAuthenticator struct {
-	q           pg.Q
+	q           pg.Q //TODO replace
```

Comment:
Probably needs a ticket number for tracking

Reply:
Ah I meant to circle back to this one but the tests started passing before I did. Will update

Reply:
Updated